ember-file-upload/src/helpers/file-queue.ts

```diff
@@ -36,9 +30,9 @@ interface FileQueueArgs {
  * ```
  */
 export default class FileQueueHelper extends Helper implements QueueListener {
-  @service declare fileQueue: FileQueueService;
+  @service fileQueue!: FileQueueService;
```

Comment:
Replaced the declare keyword here because the default typescript transpiler complained about it 🤷🏻

Reply:
Hum, I'd figure out why it's complaining. I think we want `declare`

Reply:
Reverted to using `declare`. Configured with the following in babel config.
```
["@babel/plugin-transform-typescript", { "allowDeclareFields": true }],
```